recipes/spyder-line-profiler/run_test.py

```diff
@@ -0,0 +1,9 @@
+"""
+Test whether spyder_line_profiler is installed
+
+The test is only whether the module can be found. It does not attempt
+to import the module because this needs an X server.
+"""
+
+import imp
+imp.find_module('spyder_line_profiler')
```

Comment:
You can add the necessary X server deps using `yum_requirements.txt` if you want to perform that test. But this is fine too, just add a not to why you took this route.